src/components/Simulering/utils.ts

```diff
@@ -50,32 +50,38 @@ export const processPensjonsberegningArray = (
   return dataArray
 }
 
+// TODO PEK-163 håndtere at en avtale ikke alltid har startaar - hva gjør vi da?
```

Comment:
@espengjostol angående disse to punktene her:
- avtale uten startAar -> fanger du dem i backend til vi får et bedre svar fra NP eller skal jeg legge en sjekk her?
-  sluttMaaned lik eller lavere startMaaned -> det bør i utgangspunktet ikke skje, men "in case", har du en guard mot det eller skal jeg ha noe her? (hvis ikke kan frontend plutselig vise "67 år og -3md.")

Reply:
- avtale uten startAar -> Ja, de filtreres nå ut i backend.
- sluttMaaned lik eller lavere startMaaned -> Backend har ingen "guard" mot det. Men jeg skjønner ikke behovet. Hvis vi f.eks. har
```
                  startAlder: { aar: 62, maaneder: 11 },
                  sluttAlder: { aar: 64, maaneder: 8 },
```
så viser frontend –3 md.? Hvorfor?

Reply:
- avtale uten startAar -> Kan det oppdateres i Swagger slik at Scheme stemmer med at startAar alltid er med?
- sluttMaaned lik eller laverer startMaaned -> Her poengterte jeg tilfeller i samme år (det er greit når det er som du sier fra 62,11 til 64,8, men ikke hvis det er f.eks 62,11 til 62,8):

```
// TODO PEK-163 Bør legges til en guard slik at sluttmaaned ikke får lov til å være lik eller lavere enn startMaaned når det er på samme år
//  expect(
//   getAntallMaanederMedPensjon(
//     67,
//     { aar: 67, maaneder: 3 },
//     { aar: 67, maaneder: 3 }
//   )
// ).toBe(1)
```

Reply:
- avtale uten startAar -> Ja, oppdaterer i Swagger
- sluttMaaned lik eller laverer startMaaned -> Ok, legger til sjekk i backend på at startalder må være mindre enn eller lik sluttalder